src/node/util.ts

```diff
@@ -20,19 +22,23 @@ export const paths = getEnvPaths()
  * On MacOS this function gets the standard XDG directories instead of using the native macOS
  * ones. Most CLIs do this as in practice only GUI apps use the standard macOS directories.
  */
-function getEnvPaths(): Paths {
+export function getEnvPaths(): Paths {
   let paths: Paths
   if (process.platform === "win32") {
-    paths = envPaths("code-server", {
-      suffix: "",
-    })
+    paths = {
+      ...envPaths("code-server", {
+        suffix: "",
+      }),
+      runtime: tmpdir,
+    }
```

Comment:
I think this is fine, but would like a second opinion.

Reply:
`xdg-basedir` is not meant to be used on Windows (or technically on Mac according to the docs haha but we use it anyway) so maybe we can use `temp` instead since there doesn't seem to be a runtime concept.

Edit: see suggestion below